examples/storybooks/sandbox-utils.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { getParameters } from 'codesandbox/lib/api/define';
+
+const GIT_URL =
+  'https://api.github.com/repos/fritz-c/react-sortable-tree/contents';
+
+export const SANDBOX_URL = 'https://codesandbox.io/api/v1/sandboxes/define';
+
+// full url for github api call
+const getURL = filename => `${GIT_URL}/examples/storybooks/${filename}`;
+
+// strip ../../src from the src
+const strip = code => code.replace('../../src', 'react-sortable-tree');
+
+// modify code so we can just have one file in the sandbox. index.js
+const modify = code => {
+  const addToTop = `import { render } from 'react-dom';\n`;
+  let addToBottom = `\nrender(<App />, document.getElementById('root'));`;
+  const newBottom = addToBottom.replace('App', 'NewApp');
+
+  // in some files the App component is wrapped with DragDropContext so we have to replace it
+  if (code.includes(`DragDropContext(HTML5Backend)(App)`)) {
+    addToBottom = `\nconst NewApp = DragDropContext(HTML5Backend)(App);${newBottom}`;
+  } else if (code.includes(`DragDropContext(isTouchDevice`)) {
+    addToBottom = `\nconst NewApp = DragDropContext(isTouchDevice ? TouchBackend : HTML5Backend)(App);${newBottom}`;
+  }
+  return addToTop + code + addToBottom;
```

Comment:
A little hacky because we export a wrapped component

Reply:
Go ahead and modify the example files so they declare the class as “unwrapped” or something, set the wrapped component to “const App”, and export that as the default. Wouldn’t hurt to add some convention like that for the example files.

Reply:
Sounds good!

Reply:
Sounds good!